examples/workflow_example_cep.jl

```diff
@@ -2,15 +2,15 @@
 using CapacityExpansion
 using Clp
 ## LOAD DATA ##
-state="GER_18" # or "GER_18" or "CA_1" or "TX_1"
-years=[2015] #2016 works for GER_1 and CA_1, GER_1 can also be used with 2006 to 2016 and, GER_18 is 2015 TX_1 is 2008
+state="CA_14" # or "GER_18" or "CA_1" or "TX_1"
```

Comment:
Is this supposed to be a long term change that workflow examples are with CA or just left over from testing?

Reply:
That was testing... reversed it.